scip-sys/build.rs

```diff
@@ -54,6 +54,10 @@ fn lib_scip_in_dir(path: &str) -> bool {
 }
 
 fn main() -> Result<(), Box<dyn Error>> {
+    if std::env::var("DOCS_RS").is_ok() {
+        return Ok(()); // Don't build on docs.rs
+    }
```

Comment:
I don't think this is the right solution.
You won't have any documentation.
You have to be able to build on docs.rs

See https://forge.rust-lang.org/docs-rs/add-dependencies.html

Since it is not readily available as a package on ubuntu, maybe the best solution is to bundle the source code of scip as a git subrepo, and to build it when building russcip if it is not already available on the system.

This would also make installation easier for everyone, remove the dependency to python and to the conda package manager.

Reply:
Thank you for sharing this. This would definitely be nice but I would rather work on adding more features now so it can be used more extensively than make the installation easier. Especially since building SCIP from source can be quite challenging sometimes. Anyways,  I will add it as an issue and I can also link to the public SCIP C-API in `scip-sys`'s README.

Reply:
Added #25 & #24.